demos/python_demos/handwritten_japanese_recognition_demo/handwritten_japanese_recognition_demo.py

```diff
@@ -0,0 +1,135 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from __future__ import print_function
+import os
+import sys
+import time
+import logging as log
+from argparse import ArgumentParser, SUPPRESS
+
+import cv2
+import numpy as np
+
+from openvino.inference_engine import IENetwork, IECore
+from utils.codec import CTCCodec
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument("-m", "--model", type=str, required=True,
+                      help="Path to an .xml file with a trained model.")
+    args.add_argument("-i", "--input", type=str, nargs="+", required=False,
+                      help="Required. Path to an image to infer")
+    args.add_argument('-ih', '--input-height', default=96, type=int,
+                      help='IR input image height')
+    args.add_argument('-iw', '--input-width', default=2000, type=int,
+                      help='IR input image width')
+    args.add_argument("-d", "--device", type=str, default="CPU",
+                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL, MYRIAD or HETERO: is "
+                           "acceptable. The sample will look for a suitable plugin for device specified. Default "
+                           "value is CPU")
+    args.add_argument("-ni", "--number_iter", type=int, default=1,
+                      help="Optional. Number of inference iterations")
+    args.add_argument("-lang", "--lang", type=str, default='jp',
```

Comment:
I can not understand this parameter appointment. Is there the difference in format of charlist which depends on language?

Reply:
No. Currently the model only supports Japanese. This parameter can be deleted.